cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/MonitorConfig.java

```diff
@@ -311,9 +312,18 @@ private MonitorConfig() {
   public static final String SAMPLE_STORE_CLASS_CONFIG = "sample.store.class";
   public static final String DEFAULT_SAMPLE_STORE_CLASS = KafkaSampleStore.class.getName();
   public static final String SAMPLE_STORE_CLASS_DOC = "The sample store class name. User may configure a sample store "
-      + "that persist the metric samples that have already been aggregated into Kafka Cruise Control. Later on the "
+      + "that persists the metric samples that have already been aggregated into Kafka Cruise Control. Later on the "
       + "persisted samples can be reloaded from the sample store to Kafka Cruise Control.";
 
+  /**
+   * <code>sample.partition.metrics.store.ongoing.execution.class</code>
+   */
+  public static final String SAMPLE_PARTITION_METRIC_STORE_ONGOING_EXECUTION_CLASS_CONFIG = "sample.partition.metric.store.ongoing.execution.class";
+  public static final String DEFAULT_SAMPLE_PARTITION_METRIC_STORE_ONGOING_EXECUTION_CLASS = NoopSampleStore.class.getName();
+  public static final String SAMPLE_PARTITION_METRIC_STORE_ONGOING_EXECUTION_CLASS_DOC = "The sample store during execution class name. "
+      + "User may configure a sample store that persists the partition metric samples collected while there is an ongoing execution. "
+      + "This config is not intended for enhancing the fault tolerance of the system through recovery of the historical load information.";
+
```

Comment:
Since some of the config names have changed from `ongoing` to `on`, should we use consistent name replacement from `ongoing` to `on` in all config names?